tests/model_selection/test_keras.py

```diff
@@ -0,0 +1,76 @@
+import pickle
+from typing import Tuple
+
+import numpy as np
+import pandas as pd
+import pytest
+from distributed.utils_test import gen_cluster
+from scipy.stats import loguniform, uniform
+from sklearn.base import clone
+from sklearn.datasets import make_classification, make_regression
+from sklearn.exceptions import DataConversionWarning
+from sklearn.model_selection import RandomizedSearchCV
+
+from dask_ml.model_selection import IncrementalSearchCV
+
+try:
+    import tensorflow as tf
+    from tensorflow.keras.datasets import mnist as keras_mnist
+    from tensorflow.keras.layers import Activation, Dense, Dropout
+    from tensorflow.keras.models import Sequential
+    from tensorflow.keras.utils import to_categorical
+    from scikeras import KerasClassifier
+except ImportError:
+    pytestmark = pytest.mark.skip(reason="Missing tensorflow or scikeras")
+
+pytest.importorskip("tensorflow", "2.3.0", reason="pickle support")
+
+
+def mnist() -> Tuple[np.ndarray, np.ndarray]:
+    (X_train, y_train), _ = keras_mnist.load_data()
+    X_train = X_train[:100]
+    y_train = y_train[:100]
+    X_train = X_train.reshape(X_train.shape[0], 784)
+    X_train = X_train.astype("float32")
+    X_train /= 255
+    Y_train = to_categorical(y_train, 10)
+    return X_train, y_train
+
+
+def _keras_build_fn(lr=0.01):
+    layers = [
+        Dense(512, input_shape=(784,), activation="relu"),
+        Dense(10, input_shape=(512,), activation="softmax"),
+    ]
+
+    model = Sequential(layers)
+
+    opt = tf.keras.optimizers.SGD(learning_rate=lr)
+    model.compile(loss="categorical_crossentropy", optimizer=opt, metrics=["accuracy"])
+    return model
+
+
+@gen_cluster(client=True)
```

Comment:
```suggestion
@gen_cluster(client=True, Worker=Nanny)
```

I'm curious, does Keras still work if we have to cross a process boundary?  We've had issues with serialization and Keras in the past.

Reply:
Yes, at least in my testing with `LocalCluster(processes=True, n_workers=5, threads_per_worker=2)`.

This is new behavior: Dask-ML requires Tensorflow v2.3.0, which is only 3 days old.